File: HANSOLO.go
package main

import (
	"fmt"
	"os"
)

func main() {
	file, err := os.Create("HANSOLO.hs")
	if err != nil {
		fmt.Println(err)
		return
	}
	defer file.Close()

	for i := 0; i < 1000000; i++ {
		file.WriteString("HANSOLO\n")
	}
	fmt.Println("HANSOLO.hs file created successfully.")
}

File: HANSOLO.rs
use std::fs::File;
use std::io::prelude::*;

fn main() -> std::io::Result<()> {
    let mut file = File::create("HANSOLO.hs")?;
    for _ in 0..1_000_000 {
        file.write_all(b"HANSOLO\n")?;
    }
    Ok(())
}
